integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/setup/withapim/CcStartupExecutorTwo.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.choreo.connect.tests.setup.withapim;
+
+import org.awaitility.Awaitility;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.wso2.choreo.connect.tests.apim.ApimBaseTest;
+import org.wso2.choreo.connect.tests.context.CcInstance;
+import org.wso2.choreo.connect.tests.util.TestConstant;
+import org.wso2.choreo.connect.tests.util.Utils;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * This class is prepared to invoke CC for test cases that cover specific scenarios.
+ */
```

Comment:
Let's mention here what the scenarios are. Like,

1. When "Internal-key" header is also configured as "api-key"

Reply:
Added with latest commit.